docs/faq/non-mc-projects.md

```diff
@@ -0,0 +1,6 @@
+# Are non-Minecraft projects allowed?
+
+Yes.
```

Comment:
I would be explicit about what kinds of non-MC projects we accept (Minecraft clones, completely different games, corporate-level apps, etc), or at the very least, be explicit about what kinds of projects we **don't** accept.

Reply:
I don't feel like this is a necessary thing here. At most could I say it needs to be a Java project that complies with our Usage Guidelines, but either giving examples of what we accept or not accept could leave open some holes for people to later abuse... Tho, that's my view of things.

Reply:
Should hopefully be clearer now